src/python/WMComponent/AgentStatusWatcher/AgentStatusPoller.py

```diff
@@ -168,6 +169,9 @@ def collectAgentInfo(self):
             if float(disk['percent'].strip('%')) >= diskUseThreshold and \
                             disk['mounted'] not in self.config.AnalyticsDataCollector.ignoreDisk:
                 agentInfo['disk_warning'].append(disk)
+                agentInfo['drain_mode'] = True
+                self.config.WorkQueueManager.queueParams['DrainMode'] = True
```

Comment:
As mentioned in the meeting, can you check whether this configuration change gets automatically persisted in disk (config/wmagent/config.py)?